backend/docker-compose.yml

```diff
@@ -5,83 +5,68 @@ services:
     image: postgres:14
     environment:
       - POSTGRES_HOST_AUTH_METHOD=trust
-    ports:
-      - 5432:5432
-    depends_on:
-      elasticsearch:
-        condition: service_healthy
     volumes:
       - pg_data:/var/lib/postgresql/data
-    networks:
-      - ws_front
     healthcheck:
       test: ["CMD-SHELL", "pg_isready -U postgres"]
-      interval: 10s
+      interval: 1s
       timeout: 5s
       retries: 5
 
-  pg_metrics:
-    image: wrouesnel/postgres_exporter
-    networks:
-      - ws_front
+  minio:
+    image: minio/minio
+    volumes:
+      - minio_data:/data
+    # ports:
+    #   - 9001:9001
```

Comment:
Do we need this comment?

Reply:
It's the management port for Minio, which will likely be useful for managing file retention as well as backups and wrangling failed encodes. We could probably delete it. Or we could just force it to bind to `127.0.0.1` so we don't have to worry about unintended exposure of the port, which is what I was trying to avoid by commenting it out.

Reply:
Sounds good to me